### PR TITLE
Feature/tanium to oscal

### LINKED ISSUE #361: As a tanium user I would like to convert tanium spreadsheet output to OSCAL to provide a standardized setup
Tanium is an endpoint security tool that can report CIS benchmarks in a simple, yet proprietary format.

Objectives:
- Convert tanium output to oscal assessment results
- Update website to note that Tanium an Open Shift compliance operator are supported as 3rd party tools.

### PR DESCRIPTION
**Pull request by degenaro**
## Types of changes

- \[ \] Bug fix (non-breaking change which fixes an issue)
- \[x] New feature (non-breaking change which adds functionality)
- \[ \] Breaking change (fix or feature that would cause existing functionality to change)
- \[x] My code follows the code style of this project.
- \[ \] My change requires a change to the documentation.
- \[ \] I have updated the documentation accordingly.
- \[x] I have added tests to cover my changes.
- \[x] All new and existing tests passed.
- \[x] All commits are signed-off.

## Description

- Provide transformation function from Tanium report data comprising individual entries consumable as JSON to OSCAL-like Observation objects list or as JSON string, as requested by caller.
- Accept transformation timestamp from config.
- Provide trestle task tanium-to-oscal to read input OSCO files from input directory and write OSCAL-like transformation data to corresponding files in output directory.
- Provide test-cases covering transformation function and task.
- Update website to note that Tanium an Open Shift compliance operator are supported as 3rd party tools.

## Comments

The *transformation timestamp from config* feature is meant to be temporary until such time that Tanium report comprises a key "Timestamp" or equivalent.

Closes #361

### CODE REVIEW

File: trestle/tasks/tanium_to_oscal.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                # write the OSCAL to the output file
+                write_file = pathlib.Path(ofile).open('w', encoding=const.FILE_ENCODING)
+                write_file.write(observations)
+                # display analysis
+                if not quiet:
+                    logger.info(f'Rules Analysis:')
+                    logger.info(f'rules [dispatched]: {analysis["dispatched_rules"]}')
+                    logger.info(f'rules [unique]: {analysis["unique_rules"]}')
+                    logger.info(f'results: {analysis["results"]}')
+        return TaskOutcome(mode+'success')
```
Comment (butler54) [inline trestle/tasks/tanium_to_oscal.py:134]: This is super minor, however, some of the operations are missing the standard spaces from a python perspective (like line 134).

**Review by butler54 — APPROVED** [review]
LGTM - there are some minor formatting issues to clean up otherwise it's good to go.

File: trestle/tasks/tanium_to_oscal.py (hunk 1 above)
Comment (degenaro) [reply trestle/tasks/tanium_to_oscal.py]: Fixed. Not sure why make code-format, code-lint don't fix/complain in the trestle project. I copied module to arboretum project and this problem was detected and fixed there.

**Comment by butler54** [comment]
Okay I think there has been dependency change here that is causing some issues with underlying python libraries.

**Pull request merged**